### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by moise-mulungu**
On this milestone, I:
- Created an app as my entry-point
- Added the following methods to it:
- `List all books.`
- `List all people`.
- `Create a person` (teacher or student, not a plain Person).
- `Create a book`.
- `Create a rental`.
- `List all rentals for a given person id`
- Created the main file to define the entry-point
- Added a method called `main` to it
- Updated the README file.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+    # case person_type
+    # when '1'
+    #   create_student
+    # when '2'
+    #   create_teacher
+    # end
+    # puts "Person created successfully\n"
+    # gets
+    # run
```
Comment (CollinsTatang) [inline app.rb]: - [ ] Please remove all commented code from your codebase to make your codebase more readable and easy to maintain.

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+  def create_student
+    print 'Age: '
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    print 'Has parent permission? [Y/N]: '
+    permission = gets.chomp
+    student = Student.new(age, name, my_permission(permission))
+    @people.push(student)
+  end
```
Comment (CollinsTatang) [inline app.rb]: - [ ] Great job done so far, but your app still has some issues that still need to be addressed as per project requirements. The app generates an error when trying to give permission to a new student.
permission

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    when '4'
+      create_book
+    when '5'
+      create_rental
+    when '6'
+      list_rentals
+    else
+      puts 'Thanks for using the app.'
+      Exit
+    end
```
Comment (CollinsTatang) [inline app.rb]: - [ ] Please your app throws an error when users try to exit the app
exit

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @moise-mulungu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required Changes ♻️

- [ ] Please improve your README file by adding the different steps required to run your project locally.  

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 

## Hi @moise-mulungu 👋, 

Well done implementing the suggested changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Highlights ⭐ 
✔️ Descriptive PR
✔️ Correct Gitflow
✔️ Linter checks
✔️ App works as expected


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**